lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -146,6 +146,16 @@ class AwsCompileSQSEvents {
               [queueLogicalId]: JSON.parse(sqsTemplate),
             };
 
+            if (event.sqs.batchSize) {
+              _.merge(newSQSObject, {
+                [queueLogicalId]: {
+                  Properties: {
+                    MaximumRetryAttempts: event.sqs.maximumRetryAttempts,
+                  },
+                },
+              });
```

Comment:
Let's use plain JS as:

```javascript
newSQSObject[queueLogicalId].Properties.MaximumRetryAttempts = event.sqs.maximumRetryAttempts
```

(idea is to resort to lodash only if we can't achieve something cleanly with plain language constructs)

Reply:
Ok!